doc/manual/modules/performance/ref_available-tuned-plug-ins.adoc

```diff
@@ -139,38 +139,92 @@ The powersave level can be specified by using the [option]`radeon_powersave` opt
 For details, see link:http://www.x.org/wiki/RadeonFeature#KMS_Power_Management_Options[www.x.org]. Note that this plug-in is experimental and the option might change in future releases. 
 
 `bootloader`::
-Adds options to the kernel command line. This plug-in supports only the GRUB 2 boot loader.
+Adds options to the kernel command line. This plug-in supports the GRUB 2 boot loader and the Boot Loader Specification (BLS).
 +
 Customized non-standard location of the GRUB 2 configuration file can be specified by the [option]`grub2_cfg_file` option.
 +
 The kernel options are added to the current GRUB configuration and its templates. The system needs to be rebooted for the kernel options to take effect.
 +
-Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file.
+Switching to another profile or manually stopping the `tuned` service removes the additional options. If you shut down or reboot the system, the kernel options persist in the [filename]`grub.cfg` file and grub environment files.
```

Comment:
"grub" should be written GRUB, unless it's a code literal.

Reply:
Thanks for the review, fixed.